public/App.js

```diff
@@ -16,8 +15,8 @@ const ctx = canvas.getContext('2d');
 let windowWidth = window.innerWidth;
 let windowHeight = window.innerHeight;
 
-let originalWidth = 240;
-let originalHeight = 135;
+let originalWidth = 320;
```

Comment:
This can't live in `App.js` in the future, since games will ideally render at different resolutions and `App.js` shouldn't have to know every game's original resolution. The messaging between server and clients needs work (eg. partial re-rendering as opposed to whole array), so this will likely be communicated by the server to each client on initial connection.